osu.Game/Overlays/Profile/UserGraph.cs

```diff
@@ -83,7 +88,7 @@ protected override void OnHoverLost(HoverLostEvent e)
 
         private void redrawGraph()
         {
-            if (data.Length == 0)
```

Comment:
```csharp
data?.Length > 1
!data?.Any() ?? true
```
No consistency still.

Reply:
github bugged out as it seemingly doesn't like review comments of a range of commits, but the second variant is there in the current diff.